app/api/groups/get_members.go

```diff
@@ -120,25 +120,33 @@ func (srv *Service) getMembers(w http.ResponseWriter, r *http.Request) service.A
 	query := srv.Store.GroupGroups().
 		Select(`
 			groups_groups.id,
-			groups_groups.type_changed_at,
-			groups_groups.type,
+			latest_change.at AS member_since,
+			latest_change.action,
 			users.group_id AS user__group_id,
 			users.login AS user__login,
 			users.first_name AS user__first_name,
 			users.last_name AS user__last_name,
 			users.grade AS user__grade`).
 		Joins("LEFT JOIN users ON users.group_id = groups_groups.child_group_id").
+		Joins(`
```

Comment:
Isn't this service missing the membership `until` check?

Reply:
Yes, it is missing, but actually it is missing intentionally here. It's still not thought out how to remove expired relationships from `groups_groups` (while they prevent creating new relations), so I figured it would be a good idea to show them at least somewhere :/

Reply:
It means you would have to return the "until" at least. But anyway, expired members should be considered as not members anymore. I must say that I have no clear idea on how to deal with these, but the idea was to move then from this table to the "changes" table with the "expired" action... but when, I don't know yet. 
But here, I think it is better not to have them.

Reply:
Hm. It wasn't missing (notice the 'WhereGroupRelationIsActual().' at line 149). Sorry, it is not very conspicuous, I missed it too. So it is there. I added a note into the swagger docs and improved the cucumber test to check `groups_groups.expires_at`